src/base/bittorrent/session.h

```diff
@@ -367,6 +367,8 @@ namespace BitTorrent
         void setOutgoingPortsMin(int min);
         int outgoingPortsMax() const;
         void setOutgoingPortsMax(int max);
+        int uPnPLeaseDuration() const;
```

Comment:
Also I would make an exception for such names that are started with abbreviation to allow upper case at first position (instead of making them look inconsistent). E.g.: `UPnPLeaseDuration`.

Reply:
Oops! I'll fix this and the SSL PRs tonight.